Mathlib/RingTheory/MvPowerSeries/Topology.lean

```diff
@@ -0,0 +1,311 @@
+/-
+Copyright (c) 2024 Antoine Chambert-Loir, María Inés de Frutos Fernández. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Antoine Chambert-Loir, María Inés de Frutos Fernández
+-/
+
+import Mathlib.RingTheory.Nilpotent.Defs
+import Mathlib.RingTheory.MvPowerSeries.Basic
+import Mathlib.Topology.Algebra.InfiniteSum.Constructions
+import Mathlib.Topology.Algebra.Ring.Basic
+import Mathlib.Topology.Algebra.UniformGroup
+import Mathlib.Topology.UniformSpace.Pi
+
+/-! # Topology on power series
+
+Let `α` be with Semiring α` and `TopologicalSpace α`
```

Comment:
Would you mind renaming alpha to something that sounds a bit more like a topological ring? Like `R` or `A`?

Reply:
No I don't. (Actually, I had get acquainted to these greek letters in mathlib3, and this comes out former lean3 code…)